### PR TITLE
Livesync: Include subcontributions in the initial export

### PR DESCRIPTION
**Pull request by plourenco**
🚗💨

### CODE REVIEW

File: livesync/indico_livesync/initial.py [hunk 1]
```diff
@@ -89,6 +89,48 @@ def query_contributions():
...
+    session_block_strategy.joinedload(contrib_block.own_venue)
+    session_block_strategy.joinedload(contrib_block.own_room).options(raiseload('*'), joinedload('location'))
+
+    return (
+        SubContribution.query
+        .join(Contribution)
+        .join(Contribution.event.of_type(contrib_event))
+        .outerjoin(Contribution.session.of_type(contrib_session))
+        .outerjoin(Contribution.session_block.of_type(contrib_block))
+        .filter(~SubContribution.is_deleted & ~Contribution.is_deleted & ~contrib_event.is_deleted)
```
Comment (ThiefMaster) [inline livesync/indico_livesync/initial.py:122]:
```suggestion
        .filter(~SubContribution.is_deleted, ~Contribution.is_deleted, ~contrib_event.is_deleted)
```

no technical difference but I think we usually do is that way for simple AND-linked criteria

File: livesync/indico_livesync/uploader.py [hunk 2]
```diff
@@ -48,17 +48,17 @@ def run(self, records):
             self.processed_records(batch)
         self.logger.info('%s finished', self_name)
 
-    def run_initial(self, events, total=None):
+    def run_initial(self, records, total=None):
```
Comment (ThiefMaster) [inline livesync/indico_livesync/uploader.py:51]: I think we can make `total` mandatory..

Comment (plourenco) [reply livesync/indico_livesync/uploader.py:51]: Previously I would say we could want to run an export without progress. 
But right now, I don't see any cases for making it optional either.

Comment (ThiefMaster) [reply livesync/indico_livesync/uploader.py:51]: yeah, I think if we ever find a reason why we wouldn't want a progress display, that should be made explicit through a new CLI arg. but it doesn't seem particularly useful...

**Pull request merged**